docs/runsphinx.py.in

```diff
@@ -128,6 +130,11 @@ def update_path(mantidpath):
     if gui:
         return
 
+    # add 'bin' if the subdirectory exists and isn't named
```

Comment:
While this works well for Linux it won't help for Visual Studio or XCode. If we're using `mantidpython` then it already sets up the local python path doesn't it? The problem is that this script falls over if `mantidpath` is `None`.

If we changed `if gui` to `if mantidpath is not None or gui:` that would have the same affect but it would also work on all platforms as it just requires the user to call the appropriate `mantidpython` script for their build configuration. Would that work?